learning_cp/nqueens/features.jl

```diff
@@ -2,24 +2,24 @@ struct BetterFeaturization <: SeaPearl.AbstractFeaturization end
 
 function SeaPearl.featurize(sr::SeaPearl.DefaultStateRepresentation{BetterFeaturization})
     g = sr.cplayergraph
-    features = zeros(Float32, nv(g), nqueens_generator.board_size+3)
+    features = zeros(Float32, nqueens_generator.board_size+3, nv(g))
     for i in 1:nv(g)
         cp_vertex = SeaPearl.cpVertexFromIndex(g, i)
         if isa(cp_vertex, SeaPearl.VariableVertex)
             if isa(cp_vertex.variable, SeaPearl.IntVarViewOffset)
-                features[i, 1] =0
+                features[1, i] =0
             else
                 id = cp_vertex.variable.id
-                features[i, 1] = string_to_queen(id)/nqueens_generator.board_size
+                features[1, i] = string_to_queen(id)/nqueens_generator.board_size
```

Comment:
Why do we use this kind of encoding instead of a one-hot encoding on variables that could maybe carry more information (or make it more understandable by the agent) ?

Reply:
A one one vector should not be of size nqueens_generator.board_size with values exclusively in {0,1} ? Here there is only one value that can take values between 0, 0.1, 0.2 ... 1 am I wrong ?

Reply:
you're right ! but I don't think that it matters here